### ISSUE TITLE
5.0.0-beta.1 crashes on Minecraft 21w10a+

### ISSUE THREAD

**Issue by XxPKBxX** [EnnuiL/OkZoomer#22]
Crash when opening config menu on Mod Menu on 21w18a.

```
---- Minecraft Crash Report ----
// You should try our sister game, Minceraft!

Time: 5/9/21 4:51 PM
Description: Rendering screen

java.lang.NoSuchMethodError: com.mojang.blaze3d.systems.RenderSystem.color4f(FFFF)V
	at me.shedaniel.clothconfig2.gui.AbstractConfigScreen.overlayBackground(AbstractConfigScreen.java:373)
	at me.shedaniel.clothconfig2.gui.AbstractConfigScreen.overlayBackground(AbstractConfigScreen.java:364)
	at me.shedaniel.clothconfig2.gui.GlobalizedClothConfigScreen.method_25394(GlobalizedClothConfigScreen.java:205)
	at net.minecraft.class_757.method_3192(class_757.java:867)
	at net.minecraft.class_310.method_1523(class_310.java:1087)
	at net.minecraft.class_310.method_1514(class_310.java:705)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at me.shedaniel.clothconfig2.gui.AbstractConfigScreen.overlayBackground(AbstractConfigScreen.java:373)
	at me.shedaniel.clothconfig2.gui.AbstractConfigScreen.overlayBackground(AbstractConfigScreen.java:364)
	at me.shedaniel.clothconfig2.gui.GlobalizedClothConfigScreen.method_25394(GlobalizedClothConfigScreen.java:205)

-- Screen render details --
Details:
	Screen name: me.shedaniel.clothconfig2.gui.GlobalizedClothConfigScreen
	Mouse location: Scaled: (945, 54). Absolute: (1891.000000, 108.000000)
	Screen size: Scaled: (960, 540). Absolute: (1920, 1080). Scale factor of 2.000000

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, §b§l§oSimple §fInventory, Fabric Mods
Stacktrace:
	at net.minecraft.class_6360.method_36565(class_6360.java:49)
	at net.minecraft.class_310.method_1587(class_310.java:2280)
	at net.minecraft.class_310.method_1514(class_310.java:724)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

-- System Details --
Details:
	Minecraft Version: 21w18a
	Minecraft Version ID: 21w18a
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2549654264 bytes (2431 MB) / 4060086272 bytes (3872 MB) up to 4294967296 bytes (4096 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-config2: Cloth Config v5 5.0.0
		fabric: Fabric API 0.34.2+1.17
		fabric-api-base: Fabric API Base 0.3.0+8d89ed5432
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.0.2+413ddf6432
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.8+413ddf6432
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+92519afa32
		fabric-command-api-v1: Fabric Command API (v1) 1.1.1+bb68760032
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+92519afa32
		fabric-containers-v0: Fabric Containers (v0) 0.1.12+41a6855232
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.2+23b3c92e32
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+92519afa32
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.10+f08a69e132
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.1.0+8d89ed5432
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.8+d748de6c32
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+92519afa32
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+92519afa32
		fabric-item-api-v1: Fabric Item API (v1) 1.2.4+23b3c92e32
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.7+12865e7832
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+524a2e8532
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.2+36b77c3e32
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.4+23b3c92e32
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.4+92519afa32
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+92519afa32
		fabric-models-v0: Fabric Models (v0) 0.3.0+ecfd213c32
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.8+daa3672132
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.11+413ddf6432
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+92519afa32
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.10.7+f08a69e132
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.3+92519afa32
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+92519afa32
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.9+f08a69e132
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.2+12865e7832
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.8+413ddf6432
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.0+12865e7832
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+92519afa32
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+92519afa32
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+92519afa32
		fabric-rendering-v1: Fabric Rendering (v1) 1.6.0+12865e7832
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.6+12865e7832
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.1+41a6855217
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.8+23b3c92e32
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.8+23b3c92e32
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.3+41a6855232
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+92519afa32
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.10+f5d458fc32
		fabricloader: Fabric Loader 0.11.3
		fiber: fiber 0.23.0-2
		java: Java HotSpot(TM) 64-Bit Server VM 8
		libzoomer: LibZoomer 0.1.0+21w03a
		minecraft: Minecraft 1.17-alpha.21.18.a
		modmenu: Mod Menu 2.0.0-beta.4
		okzoomer: Ok Zoomer 5.0.0-beta.1+21w03a
	Launched Version: fabric-loader-0.11.3-21w18a
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 645/PCIe/SSE2 GL version 3.2.0 NVIDIA 456.71, NVIDIA Corporation
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, file/§b§l§oSimple §fInventory
	Current Language: English (Canada)
	CPU: 8x Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz
```

**Comment by XxPKBxX** [comment]
Error on launcher says:
```
The game crashed whilst rendering screen
Error: java.lang.NoSuchMethodError: com.mojang.blaze3d.systems.RenderSystem.color4f(FFFF)V
Exit Code: -1
```

Installed mods:
![image](https://user-images.githubusercontent.com/29917385/117564468-20d9a300-b0e7-11eb-89de-9d717bd41a13.png)

**Comment by XxPKBxX** [comment]
I am not sure if this error is happening because of Mod Menu, or this mod.

**Comment by joaoh1** [comment]
Due to the rendering overhaul from Minecraft 21w10a, the config screen (and iirc, the zoom) won't work at all. The indev branch should fix the zoom and the next update on Ok Zoomer will redo the entire config system on another config API, however, don't expect anything soon yet

**Comment by EnnuiL** [comment]
The second 5.0.0 beta has been released for the complete 1.17 release
